flutter-idea/src/io/flutter/sdk/FlutterSdkVersion.java

```diff
@@ -102,6 +102,9 @@ public class FlutterSdkVersion implements Comparable<FlutterSdkVersion> {
   @NotNull
   private static final FlutterSdkVersion MIN_STABLE_MACOS_PLATFORM = new FlutterSdkVersion("3.0.0");
 
+  @NotNull
+  private static final FlutterSdkVersion MIN_MODERN_DEVTOOLS_URL = new FlutterSdkVersion("3.3.0");
```

Comment:
It may be worth a comment here (or renaming) so it's clear what "modern" means just in case things change again in future and there's a "new modern" format 😄

(in VS Code I named it `supportsDartDevToolsPathUrls` since it's when we use the `PathUrlStrategy`).

Reply:
Thanks for the naming suggestion